controllers/userStatus.js

```diff
@@ -0,0 +1,99 @@
+const userStatusModel = require("../models/userStatus");
+
+/**
+ * Creates a new User Status
+ * @param req {object} - Express Request Object
+ * @param res {object} - Express Response Object
+ */
+
+const addUserStatus = async (req, res) => {
+  try {
+    const { id, userStatusData } = await userStatusModel.addUserStatus({ ...req.body });
+    return res.status(201).json({
+      message: "Status created successfully",
+      id,
+      ...userStatusData,
+    });
+  } catch (error) {
+    logger.error(`Error while creating new User Status: ${error}`);
+    return res.boom.badImplementation("An internal server error occurred");
+  }
+};
+/**
+ * Deletes a new User Status
+ *
+ * @param req {object} - Express Request Object
+ * @param res {object} - Express Response Object
+ *
+ */
+
+const deleteUserStatus = async (req, res) => {
+  try {
+    const deletedUserStatus = await userStatusModel.deleteUserStatus(req.params.userId);
+    return res.status(201).json({
+      message: "UserStatus Deleted successfully!",
+      ...deletedUserStatus,
+    });
+  } catch (error) {
+    logger.error(`Error while creating new User Status: ${error}`);
+    return res.boom.badImplementation("An internal server error occurred");
+  }
+};
+
+/**
+ * Collects single User Status
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const getUserStatus = async (req, res) => {
+  try {
+    const userData = await userStatusModel.getUserStaus(req.params.userId);
```

Comment:
You can destructure the `req.params`. It will be helpful if we want to pass more params in future